examples/features/metadata_interceptor/client/main.go

```diff
@@ -0,0 +1,88 @@
+/*
+ *
+ * Copyright 2018 gRPC authors.
```

Comment:
Update the year on the copyright notice please.

Reply:
Sorry for that, I will update it.